lib/capistrano/slack_notify.rb

```diff
@@ -96,7 +96,11 @@ def repository
     end
 
     def rev
-      @rev ||= `git ls-remote #{repository} #{branch}`.split(" ").first
+      if fetch(:scm) == :subversion
+        $rev ||= `svn info https://svn.suran.com/wmt/trunk/`.match(/^Revision: (\d+)$/)[1]
```

Comment:
Looks like we'd need a new variable for `https://svn.suran.com/wmt/trunk/` since that wouldn't be the same for everyone.